bash_completion

```diff
@@ -251,6 +251,28 @@ _comp_dequote()
     fi
 }
 
+# Try to reconstruct an incomplete word and apply _comp_dequote.
+# @param    $1  String to be expanded.  The same as _comp_dequote, but
+#               incomplete backslash, single quotation, and double quotation
+#               are allowed.
+# @var[out] REPLY  Result.  The same as _comp_dequoet.
+# @since 2.17
+_comp_dequote_incomplete()
```

Comment:
Maybe a unit test for this will be nice?

Reply:
Thanks for the suggestion! I added tests in dcf662b5263986870eef4c883602a0f1e279672d.